README.md

```diff
@@ -68,14 +68,15 @@ Follow these steps to add Stormpath user authentication to your Express.js app.
 
   **If your app is a traditional website:**
 
-  Set `website` to `true`. This will setup default views for login, registration, etc.
+  Initialize the Stormpath module, and pass an empty set of options:
 
   ```javascript
-  app.use(stormpath.init(app, {
-    website: true
-  }));
+  app.use(stormpath.init(app, { }));
   ```
 
+  This will enable the default features, such as login and registration pages
```

Comment:
Add a period?

*(I will just comment here and fix them myself when I'm done :) )*